### DISCUSSION TITLE
Requesting Code Reviews to Other user

### DISCUSSION THREAD

**Discussion by sleeky0x0** [community/community#137707] [Programming Help] [answered]
### Body

I have recently contributed a new feature/fix to our project and would like to request a code review from my supervisor.
But once I click on reviewer button I couldn't search github users like inviting collaborators.
Am I missing something here?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Azymack** [comment, marked as answer]
it doesn't make sense.
Only collaborators are able to be invited as code reviewer.
